allauth/socialaccount/admin.py

```diff
@@ -13,7 +13,8 @@ class Meta:
         widgets = {
             'client_id': forms.TextInput(attrs={'size': '100'}),
             'key': forms.TextInput(attrs={'size': '100'}),
-            'secret': forms.TextInput(attrs={'size': '100'})
+            'secret': forms.TextInput(attrs={'size': '100'}),
+            'cert': forms.Textarea(),
```

Comment:
Can you please update `docs/providers.rst` instructions too?

Reply:
Done that now